wandb/apis/internal.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, default_settings=None, load_settings=True, retry_timedelta=da
                 # this timeout won't apply when the DNS lookup fails. in that case, it will be 60s
                 # https://bugs.python.org/issue22889
                 timeout=self.HTTP_TIMEOUT,
-                auth=("api", self.api_key),
+                auth=("api", self.api_key or ""),
```

Comment:
I was getting deprecation warnings in the latest requests lib when this is None.